### ISSUE TITLE
SP Page Builder Update Fails with "Failed to Download Package" (403 Error) on Joomla 3.10

### ISSUE THREAD

**Issue by Harmeen928** [joomla/joomla-cms#48160]
### Steps to reproduce the issue

1.Warning
Error connecting to the server: 403
Before updating ensure that the update is compatible with your Joomla! installation.
You are strongly advised to make a backup of your site's files and database before you start updating.
×
Error
Failed to download package. Download it and install manually from https://www.joomshaper.com/index.php?option=com_product&task=download.download&package=release&id=10381.

### Version

5.4
